src/test/java/ApplicationTest.java

```diff
@@ -0,0 +1,43 @@
+import static camp.nextstep.edu.missionutils.test.Assertions.assertRandomNumberInRangeTest;
+import static camp.nextstep.edu.missionutils.test.Assertions.assertSimpleTest;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+class ApplicationTest extends NsTest {
+    @DisplayName("사다리 생성 결과 출력")
+    @Test
+    void testLadderGameResult() {
+        assertRandomNumberInRangeTest(
+                () -> {
+                    run("pobi,honux,crong,jk", "5");
+                    assertThat(output()).contains(
+                            "실행결과",
+                            " pobi honux crong    jk",
+                            "    |-----|     |-----|\n"
+                                    + "    |     |-----|     |\n"
+                                    + "    |-----|     |     |\n"
+                                    + "    |     |-----|     |\n"
+                                    + "    |-----|     |-----|"
+                    );
```

Comment:
```suggestion
                    assertThat(output()).contains(
                            "실행결과",
                            " pobi honux crong    jk\n",
                            "    |-----|     |-----|\n",
                            "    |     |-----|     |\n",
                            "    |-----|     |     |\n",
                            "    |     |-----|     |\n",
                            "    |-----|     |-----|"
                    );
```
소소한 팁인데요, 이렇게 보기 좋게 바꿔보는건 어떨까요? :)

Reply:
말씀해주신 대로 출력문 가독성까지 고려하면 더 좋을 것 같아요🤔
그런데, 사다리 순서를 바꿨을 때도 테스트가 통과되는 문제가 있는 것 같습니당!!

Reply:
아 `contains` 군요...! `containsSubsequence` 를 사용하면 우리가 원하는 요소들만 특정해서 그 순서도 검증할 수 있어요 :)
```suggestion
                    assertThat(output()).containsSubsequence(
                            "실행결과",
                            " pobi honux crong    jk\n",
                            "    |-----|     |-----|\n",
                            "    |     |-----|     |\n",
                            "    |-----|     |     |\n",
                            "    |     |-----|     |\n",
                            "    |-----|     |-----|"
                    );
```
근데 생각해보니 그 사이에 다른 줄이 들어가도 통과가 될 수 있긴하네요. 🤔
이건 이런 메서드도 있다는 느낌으로 봐주세요!